.github/workflows/test-build-rpm.yml

```diff
@@ -190,35 +187,87 @@ jobs:
           sudo mkdir /home/repo
           sudo chmod 777 /home/repo
           sudo chmod 777 /etc/elasticsearch/elasticsearch.yml    
-          sudo /usr/share/elasticsearch/bin/elasticsearch-plugin remove opendistro_security
           sudo sed -i '/path.logs/a path.repo: ["/home/repo"]' /etc/elasticsearch/elasticsearch.yml 
-          sudo sed -i /^opendistro_security/d /etc/elasticsearch/elasticsearch.yml
-          sudo sed -i /CN=kirk/d /etc/elasticsearch/elasticsearch.yml
           sudo sed -i /^node.max_local_storage_nodes/d /etc/elasticsearch/elasticsearch.yml
-          sudo sed -i /^cluster.routing.allocation.disk.threshold_enabled/d /etc/elasticsearch/elasticsearch.yml
           sudo systemctl start elasticsearch.service
           sleep 30
-
-  Test-All-Plugins:
-    needs: [Start-ES-Service]
+          
+      - name: Retrieve plugin tags
+        run: |
+          echo ::set-env name=p_tag_ism::$(.github/scripts/plugin_tag.sh opendistro-for-elasticsearch/index-management)
+          echo ::set-env name=p_tag_alerting::$(.github/scripts/plugin_tag.sh opendistro-for-elasticsearch/alerting)
+          echo ::set-env name=p_tag_sql::$(.github/scripts/plugin_tag.sh opendistro-for-elasticsearch/sql)
+          echo ::set-env name=p_tag_knn::$(.github/scripts/plugin_tag.sh opendistro-for-elasticsearch/k-NN)
+          echo ::set-env name=p_tag_ad::$(.github/scripts/plugin_tag.sh opendistro-for-elasticsearch/anomaly-detection)
+      - name: Checking out SQL repo
+        if: always()
+        uses: actions/checkout@v1
+        with:
+          repository: opendistro-for-elasticsearch/sql
+          ref: ${{env.p_tag_sql}}
+      
+      - name: IT for SQL
+        if: always()
+        run: |
+          curl -XGET https://127.0.0.1:9200 -u admin:admin --insecure
+          export PATH=$PATH:$JAVA_HOME
+          cd $GITHUB_WORKSPACE/../sql
+          ./gradlew integTest -Dtests.rest.cluster=localhost:9200 -Dtests.cluster=localhost:9200 -Dtests.clustername=es-integrationtest -Dhttps=true -Duser=admin -Dpassword=admin
+          
+      - name: Checking out AD repo
+        if: always()
+        uses: actions/checkout@v1
+        with:
+          repository: opendistro-for-elasticsearch/anomaly-detection
+          ref: master
```

Comment:
any reason to use master here?

Reply:
Seems like he forgot to use the ad tag above

Reply:
Good catch. Use master for testing as AD 1.9.0.0 doesn't support security IT. Forgot to remove it. Fixed it.